frontend/src/modules/Traces/TraceFilter.tsx

```diff
@@ -40,6 +40,11 @@ interface TagKeyOptionItem {
 	tagCount: number;
 }
 
+interface ISpanKind {
+	label: string;
```

Comment:
We can create a const with the possible values like 
```
const SPAN_KIND = "SERVER" | "CLIENT"
```
and can strictly type the label to be `typeof SPAN_KIND` 

See if you can optimize this a bit.

Reply:
Check now if that's what you wanted